LG-14609 Move Threatmetrix response body to its own event

jmhooper opened the pull request:
We use Threatmetrix for device profiling. When a user's device profiling transaction represents a `review` or `reject` status they need to undergo the fraud review process. Part of that process is looking at what was returned by Threatmetrix in the logged response body.

The Threatmetrix response body is incredibly large. It causes some issues with Cloudwatch parsing the JSON written to the log.

This commit addresses the above by moving the Threatmetrix response into its own event. This change will allow us to continue inspecting the response body for fraud review but remove the issues with JSON parsing on the main proofing result event.

jmhooper commented:
I am marking this change as blocked. The fraud team has a dependency on the shape of this log event. They will need to make a change upstream before we can merge this. (cc @rebecca-e-wright)

matthinz reviewed (approved):
Looks good to me. I was trying to think if we want to include anything else in the response body event, but I don't think so.

jmhooper commented:
We changed directions here. I moved the bulk of the changes to #11290. I am going to come back around in a separate PR to stop logging the response body on this event.
